kymatio/scattering2d/core/scattering2d.py

```diff
@@ -28,11 +29,10 @@ def scattering2d(x, pad, unpad, backend, J, L, phi, psi, max_order):
     S_0 = unpad(S_0)
 
     output_shape = (x.shape[0], output_size) + S_0.shape[-2:]
-    print(output_shape)
 
     S = empty_like(x, output_shape)
+    S = squeeze(S_0, S, 0)
```

Comment:
This has to be done due to .squeeze() not being a method that works on tf tensors.

Reply:
makes sens

Reply:
Wait are we doing TF now?

Reply:
No - could we remove this commit?

Reply:
Yes I'll remove.

Reply:
I reverted the commits, but feel free to drop the original commits from the history.